content/components.md

```diff
@@ -50,7 +50,7 @@ export class CatsService {
 
 > info **Hint** To create a service using the CLI, simply execute the `$ nest g service cats` command.
 
-Our `CatsService` is a basic class with one property and two methods. The only new feature is that it uses the `@Injectable()` decorator. The `@Injectable()` decorator attaches metadata, which tells Nest that this class is a Nest provider. By the way, this example also uses a `Cat` interface, which probably looks something like this:
+Our `CatsService` is a basic class with one property and two methods. The only new feature is that it uses the `@Injectable()` decorator. The `@Injectable()` decorator attaches metadata, which declares that `CatsService`  is a class that can be managed by the Nest IoC container. By the way, this example also uses a `Cat` interface, which probably looks something like this:
```

Comment:
I want to draw attention to this line and say that I really like how you made it about the IoC container. I think that will end up being helpful. I'm thinking about adding an advanced section somewhere that goes into detail of how Typescript works with decorators and what they do, what kind of transpilation they end up causing, but for now I think this is a good way to help clarify some stuff